modules/useStandardScroll.js

```diff
@@ -27,9 +28,17 @@ export default function useStandardScroll(createHistory) {
 
   // `history` will invoke this listener synchronously, so `currentKey` will
   // always be defined.
-  function updateScroll({ key }) {
+  function updateScroll({ key, hash }) {
     currentKey = key
-
+    if (hash) {
+      // node may not yet be in DOM
+      requestAnimationFrame(() => {
+        const node = querySelectorAll(window, hash).pop()
```

Comment:
Yes, it is equivalent, but since Dom helpers was already being used I used this. Can switch if native is better.